[Kaptain] Fixed links & fine-tuning (#4368)
mesosphere/dcos-docs-site: pages/dkp/kaptain/2.0.0/install/deploy-kaptain/index.md

You can deploy Kaptain to a cluster in a selected workspace. If you do not intend to deploy Kaptain to a certain cluster, you must switch the workspace you are deploying to or move that cluster to another workspace.
</p>

## Prerequisite

> Ensure you have added Kaptain to your DKP Catalog applications before you deploy it to your clusters. Refer to the corresponding documentation depending on your environment:

Reviewer: ```suggestion
Ensure you add Kaptain to your DKP Catalog applications before you deploy it to your clusters. Refer to the corresponding documentation for your environment:
```

1.  Select **Applications** from the sidebar menu.

> 1.  Type Kaptain in the **Applications** search bar. If Kaptain is not available in the UI, add Kaptain to your DKP catalog as stated in the [prerequisites][#prerequisite].

Author: ```suggestion
1.  Type Kaptain in the **Applications** search bar. If Kaptain is not available in the UI, add Kaptain to your DKP catalog as stated in the [prerequisite section](#prerequisite).
```
